remote-execution/Part 02 - Intro to Private Training with Remote Execution.ipynb

```diff
@@ -0,0 +1,448 @@
+{
```

Comment:
<p>Also did not receive 95% accuracy.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dropoutlabs/tf-world-tutorial/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
ah good catch! forgot to update when i switch mnist to fashion mnist...